src/components/player/cue-bar.tsx

```diff
@@ -107,19 +107,23 @@ const NoteCue: React.FC<any> = ({
 }) => {
   const {setPlayerPrefs, getPlayerPrefs} = useEggheadPlayerPrefs()
   const [visible, setVisible] = React.useState(false)
+  const [clickedOpen, setClickedOpen] = React.useState(false)
   const {muteNotes} = getPlayerPrefs()
 
   useCue(cue, actions)
 
-  const open = () => {
+  const clickOpen = () => {
     setVisible(true)
+    setClickedOpen(true)
     // if we seek to the correct time, the note is displayed
     actions.seek(cue.startTime)
+    actions.pause()
```

Comment:
playing with it, I wanted it to pause when I explicitly click a cue so I can read at my leisure.

also important to note when the cue is open because of a CLICK or because of auto display,